tests/test_live_api.py

```diff
@@ -75,15 +75,26 @@ def test_get_itinerary(self):
 
     def test_get_image_url(self):
         url = self.dossier.get_map_url()
-        self.assertIsInstance(url, basestring)
+
+        if sys.version_info.major < 3:
```

Comment:
We can also check by trying to import Python 2 only modules and catching the exception, but I thought this was cleaner. Yeah, let's just keep it until we remove Python 2.